examples/advanced_operations/add_smart_display_ad.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This adds a smart display campaign, an ad group, and a responsive display ad.
```

Comment:
nit: 81 columns

Reply:
I'm seeing exactly 80, my IDE isn't complaining about it.